installer/tue-install-impl.bash

```diff
@@ -591,21 +601,21 @@ function tue-install-ros
             tue-install-target hsr-setup || tue-install-error "Failed to install target 'hsr-setup'"
         fi
 
-        tue-install-system ros-$TUE_ROS_DISTRO-$src
+        tue-install-system ros-"$TUE_ROS_DISTRO"-"$src"
         return 0
     fi
 
-    if [ -z $ROS_PACKAGE_INSTALL_DIR ]
+    if [ -z "$ROS_PACKAGE_INSTALL_DIR" ]
     then
         tue-install-error "Environment variable ROS_PACKAGE_INSTALL_DIR not set."
     fi
 
     # Make sure the ROS package install dir exists
     tue-install-debug "Creating ROS package install dir: $ROS_PACKAGE_INSTALL_DIR"
-    mkdir -p $ROS_PACKAGE_INSTALL_DIR
+    mkdir -p "$ROS_PACKAGE_INSTALL_DIR"
 
-    local ros_pkg_dir=$ROS_PACKAGE_INSTALL_DIR/$ros_pkg_name
-    local repos_dir=$TUE_REPOS_DIR/$src
+    local ros_pkg_dir="$ROS_PACKAGE_INSTALL_DIR/$ros_pkg_name"
+    local repos_dir="$TUE_REPOS_DIR/$src"
```

Comment:
Don't include `/` in quotes

Reply:
why?

Reply:
Because code highlighting on vim shows it distinctly when not in quotes :)